src/Model/Item.php

```diff
@@ -1988,6 +1991,35 @@ public static function insert($item, $notify = false, $dontcache = false)
 		return $current_post;
 	}
 
+	/**
+	 * Convert items to forum posts
+	 *
+	 * @param array $item
+	 * @return void
+	 */
+	private static function transformToForumPost(array $item)
+	{
+		if ($item["verb"] != Activity::ANNOUNCE) {
+			return;
+		}
+
+		$pcontact = Contact::selectFirst(['nurl'], ['id' => $item['author-id'], 'contact-type' => Contact::TYPE_COMMUNITY]);
+		if (empty($pcontact['nurl'])) {
+			return;
+		}
```

Comment:
We keep this condition on because it's data we need to perform the transformation. Ideally it should throw an Exception on error but logging would at least be nice.

Reply:
This line not just fetches the contact, it also checks if the contact is a forum contact. Most announces we receive are not from forum contacts and it's totally okay. It is not an error and doesn't need to be logged.

Reply:
Ah, I see. I thought the announce was specific to forum posts.